ctk/common/src/test/kotlin/org/codice/compilance/verification/profile/subject/confirmations/BearerSubjectConfirmationVerificationSpec.kt

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compilance.verification.profile.subject.confirmations
+
+import com.google.common.io.Resources
+import io.kotlintest.forAll
+import io.kotlintest.matchers.string.shouldContain
+import io.kotlintest.shouldThrow
+import io.kotlintest.specs.StringSpec
+import org.codice.compliance.Common
+import org.codice.compliance.IMPLEMENTATION_PATH
+import org.codice.compliance.SAMLComplianceException
+import org.codice.compliance.SAMLProfiles_4_1_4_2_g
+import org.codice.compliance.SAMLProfiles_4_1_4_2_h
+import org.codice.compliance.SAMLProfiles_4_1_4_2_i
+import org.codice.compliance.SAMLProfiles_4_1_4_2_j
+import org.codice.compliance.SAMLProfiles_4_1_4_2_k
+import org.codice.compliance.TEST_SP_METADATA_PROPERTY
+import org.codice.compliance.utils.BEARER
+import org.codice.compliance.utils.HOLDER_OF_KEY_URI
+import org.codice.compliance.utils.TestCommon.Companion.REQUEST_ID
+import org.codice.compliance.verification.profile.subject.confirmations.BearerSubjectConfirmationVerification
+import java.time.Instant
+import java.util.UUID
+
+@Suppress("StringLiteralDuplication")
+class BearerSubjectConfirmationVerificationSpec : StringSpec() {
```

Comment:
❓ Is there a reason you don't have any positive path tests?

Reply:
It is indirectly being tested because of all the default values I set to the response creation I'll go ahead and some explicit tests